Inkeys/IdtMain.cpp

```
@@ -1014,6 +1010,15 @@ int WINAPI wWinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, LPWSTR
 
 		// 读取配置
 		{
+		#pragma region 新配置 Test
```

Cppcheck (2.20.0)

[error] The code contains unhandled character(s) (character code=230). Neither unicode nor extended ascii is supported.

(unhandledChar)